ush/python/pygfs/task/land_analysis.py

```diff
@@ -0,0 +1,265 @@
+
+        # open tar file for writing
+        with tarfile.open(landstat, "w") as archive:
+            for diagfile in diags:
+                diaggzip = f"{diagfile}.gz"
+                archive.add(diaggzip, arcname=os.path.basename(diaggzip))
+
+        # copy full YAML from executable to ROTDIR
+        src = os.path.join(self.task_config.DATA, f"{self.task_config.CDUMP}.t{self.runtime_config.cyc:02d}z.letkfoi.yaml")
+        dest = os.path.join(self.task_config.COM_ATMOS_ANALYSIS, f"{self.task_config.CDUMP}.t{self.runtime_config.cyc:02d}z.letkfoi.yaml")
+        yaml_copy = {
+            'mkdir': [self.task_config.COM_ATMOS_ANALYSIS],
+            'copy': [[src, dest]]
+        }
+        FileHandler(yaml_copy).sync()
+
+        # ---- NOTE below is 'temporary', eventually we will not be using FMS RESTART formatted files
+        # ---- all of the rest of this method will need to be changed but requires model and JEDI changes
+        # ---- copy RESTART sfc_data files for future reference
+        template = '{}.sfc_data.tile{}.nc'.format(to_fv3time(self.task_config.current_cycle), '{tilenum}')
+        bkglist = []
+        for itile in range(1, self.task_config.ntiles + 1):
+            sfcdata = template.format(tilenum=itile)
+            src = os.path.join(self.task_config.COM_ATMOS_RESTART_PREV, sfcdata)
+            dest = os.path.join(self.task_config.COM_ATMOS_ANALYSIS, f'landges.{sfcdata}')
+            bkglist.append([src, dest])
+        FileHandler({'copy': bkglist}).sync()
+
+        # ---- add increments to RESTART files
+        # logger.info('Adding increments to RESTART files')
+        # self._add_fms_cube_sphere_increments()
+
+        # ---- move increments to ROTDIR
+        logger.info('Moving increments to ROTDIR')
+        template = f'landinc.{to_fv3time(self.task_config.current_cycle)}.sfc_data.tile{{tilenum}}.nc'
+        inclist = []
+        for itile in range(1, self.task_config.ntiles + 1):
+            sfcdata = template.format(tilenum=itile)
+            src = os.path.join(self.task_config.DATA, 'anl', sfcdata)
+            dest = os.path.join(self.task_config.COM_ATMOS_ANALYSIS, sfcdata)
+            inclist.append([src, dest])
+        FileHandler({'copy': inclist}).sync()
+
+        # ---- move analysis to ROTDIR/RESTART
+        logger.info('Moving analysis to ROTDIR')
+        template = f'{to_fv3time(self.task_config.current_cycle)}.sfc_data.tile{{tilenum}}.nc'
+        inclist = []
+        for itile in range(1, self.task_config.ntiles + 1):
+            sfcdata = template.format(tilenum=itile)
+            src = os.path.join(self.task_config.DATA, 'anl', sfcdata)
+            dest = os.path.join(self.task_config.COM_ATMOS_ANALYSIS, sfcdata)
+            inclist.append([src, dest])
+        FileHandler({'copy': inclist}).sync()
+
+    def clean(self):
+        super().clean()
+
+    @logit(logger)
+    def _add_fms_cube_sphere_increments(self: Analysis) -> None:
```

Comment:
where is this method being used?

Reply:
This `_add_fms_cube_sphere_increments` is not used in the current land DA, which can be removed.